lib/api/2.0/pollers.js

```diff
@@ -48,8 +49,16 @@ var pollersLibByIdGet = controller(function(req) {
  * @apiGroup pollers
  * @apiSuccess {json} pollers list of pollers or an empty object if there are none.
  */
-var pollersGet = controller(function(req) {
-    return pollers.getPollers(req.query);
+var pollersGet = controller(function(req, res) {
+    var options = {
+        skip: req.swagger.query.$skip,
+        limit: req.swagger.query.$top
+    };
+
+    return pollers.getPollers(req.query, options)
```

Comment:
for'req.query' maybe it should be  'var query = req.swagger ? req.swagger.query : req.query'? and extend to all query inputs?